src/Wallabag/ApiBundle/Controller/WallabagRestController.php

```diff
@@ -126,12 +126,12 @@ public function postEntriesAction(Request $request)
             $this->get('wallabag_core.content_proxy')->assignTagsToEntry($entry, $tags);
         }
 
-        if (true === (bool) $isStarred) {
-            $entry->setStarred(true);
+        if (!is_null($isStarred)) {
+            $entry->setStarred((bool) $isStarred);
         }
 
-        if (true === (bool) $isArchived) {
-            $entry->setArchived(true);
+        if (!is_null($isArchived)) {
+            $entry->setArchived((bool) $isArchived);
```

Comment:
I put them this way because you can update an entry with starred/archived at false by adding it again (#1797)